stratum/proto/openconfig/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+# Builds OpenConfig Protobuf library
+#
+# Copyright 2022-2023 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+#
+
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Same comment as above re cmake version number

Reply:
I disagree. The parent CMakeLists.txt file specifies a minimum version of 3.5, which this file inherits. I'm not aware of any features in the file that require a higher version number, which is why I omitted the command. (Bear in mind, the commands in this file were moved here from the parent file.)

We should be _removing_ unnecessary cmake_minimum_required() commands, not adding them!